gm4_blast_furnaces/data/blast_furnaces/functions/destroy.mcfunction

```diff
@@ -0,0 +1,6 @@
+#@s - @e[type=armor_stand,tag=gm4_bf_output] unless block ~ ~ ~ BLAST_FURNACE
```

Comment:
should be `unless block ~ ~ ~ hopper`

Reply:
Headers can be informal but I don't like the mix here either